### PR TITLE
Fixed the issue of interfaces not being serialized. 

### PR DESCRIPTION
**Pull request by vin-spiegel**
- Changed IPredictionInput to JObject type in Json.Net

### CODE REVIEW

**Comment by StefH** [comment]
This is strange. I would assume that JSON.NET does serialize objects with an interface.

**Comment by vin-spiegel** [comment]
yes. interface in `Request` seems to be serialized, but it seems that there is a problem when deserializing in `Prediction`.
I tried to define a type, IPredictionInput, that receives a response using generics, but I gave up because it would require changing too much codes.

**Comment by vin-spiegel** [comment]
request
```json
{
  "version": "436b051ebd8f68d23e83d22de5e198e0995357afef113768c20f0b6fcef23c8b",
  "input": {
    "prompt": "hi",
  }
}
```

response status is 400
```json
{
  "errors": {
    "input.prompt.input": [
      "Could not create an instance of type Replicate.Net.Interfaces.IPredictionInput. Type is an interface or abstract class and cannot be instantiated. Path 'input.prompt', line 4, position 13."
    ]
  },
  "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
  "title": "One or more validation errors occurred.",
  "status": 400,
  "traceId": "00-0398c9c294e06df2b21fad5981b84b23-362ba23aae10f04c-00"
}
```

**Comment by StefH** [comment]
Did you test your solution?

**Comment by vin-spiegel** [comment]
Yes

**Review by StefH — APPROVED** [review]

File: src/Replicate.Net/Models/Prediction.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public class Prediction
     /// </summary>
     public string Status { get; set; } = null!;
 
-    public IPredictionInput Input { get; set; } = null!;
+    public JObject Input { get; set; } = null!;
 
     public object? Output { get; set; }
```
Comment (StefH) [inline src/Replicate.Net/Models/Prediction.cs:32]: What do about Output?
Also change to JObject ?

Or change Input yo object?

**Review by StefH — CHANGES_REQUESTED** [review]

File: src/Replicate.Net/Models/Prediction.cs (hunk 1 above)
Comment (vin-spiegel) [reply src/Replicate.Net/Models/Prediction.cs:32]: I think Input to object is better. JObject, object both working.

**Comment by vin-spiegel** [comment]
Chaged Input JObject to object

**Review by StefH — APPROVED** [review]

**Pull request merged**